scipy/special/add_newdocs.py

```diff
@@ -3568,10 +3568,12 @@ def add_newdoc(place, name, doc):
     two rational functions of degree 6/6 and 7/7.
 
     This function is a wrapper for the Cephes [1]_ routine `j0`.
+    Not to be confused with the spherical Bessel functions (see spherical_jn).
```

Comment:
Nitpicks:

- Put backticks around `spherical_jn` to make it a link
- The addition should be a complete sentence. Something like "This function is a wrapper for the Cephes [1]_ routine `j0`. It should not be confused..."